cpp/src/community/leiden.cuh

```diff
@@ -38,22 +38,23 @@ class Leiden : public Louvain<graph_type> {
   {
     this->timer_start("update_clustering_constrained");
 
-    rmm::device_vector<vertex_t> next_cluster_v(this->cluster_v_);
+    rmm::device_vector<vertex_t> next_cluster_v(this->dendrogram_->current_level_begin(),
+                                                this->dendrogram_->current_level_end());
```

Comment:
I'm going to rework the entire Leiden code for MNMG.  I'm inclined to make this an objective in that effort rather than edit a bunch of things that might be changed in version 0.19.